README.md

```diff
@@ -9,9 +9,10 @@ These are things you will need to acquire in order to run the CFN Template in AW
 ## Slack
 You will need to generate a slack token for hubot.
 
-1. Go to https://$yourteam.slack.com/services/new/hubot and create a new hubot integration
-2. Choose a bot username, this is what you will call to the bot in rooms as (e.g.; @hubot help)
-3. Find the API Token section (the token will start with `xoxb-`)
+1. [Hubot Slack App](https://slack.com/apps/A0F7XDU93-hubot)
```

Comment:
Good point.  Do you prefer one or the other.  I like the clickable link.  I have seen these links on other tutorials however have no idea if they are permanent.